### PR TITLE
Add instructions for setting up AWS test account

### PR DESCRIPTION
**Pull request by mnitchev**
### Checklist

- [ ] Update changelog in CHANGELOG.md. _Not applicable_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +55,16 @@ This project uses [`LocalStack`](https://github.com/localstack/localstack) for i
 ```
 $ aws --endpoint=http://localhost:4566 route53 list-hosted-zones-by-name
 ```
+
+## AWS account setup
+
+To run the integration and acceptance tests against an AWS account you need to create an IAM role. To do that, first source the `aws-resolver-rules-operator-test-secrets.sh` file from LastPass and then run the following commands:
+
+```shell
+aws iam create-policy --policy-name tests-aws-network-topology-operator --policy-document file://tests/assets/test-role-policy.json
```
Comment (fiunchinho) [inline README.md:64]:
```suggestion
source aws-resolver-rules-operator-test-secrets.sh
aws iam create-policy --policy-name tests-aws-network-topology-operator --policy-document file://tests/assets/test-role-policy.json
```

**Review by fiunchinho — COMMENTED** [review]
Small suggestion, other than that LGTM

File: README.md (hunk 1 above)
Comment (fiunchinho) [inline README.md:64]: Oh another thing. Can you mention that the aws cli needs to be configured to target the AWS account found in the `WC_AWS_ACCOUNT` variable of the file?

Comment (mnitchev) [reply README.md:65]: Done

Comment (fiunchinho) [reply README.md:65]: Does the role need to exist on both MC and WC account? I think I'd mention in which one is required.

Comment (mnitchev) [reply README.md:65]: I didn't bother splitting up the permissions between the MC and WC account, but they both need permissions. For RAM in particular you need to be able to at least list the resource in the recipient account.

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**